docs/install.md

```diff
@@ -349,6 +349,10 @@ not running.
   disabled (`"false"`), which means it is disallowed to use the
   `bundle` field.
 
+- `disable-creds-init` - set this flag to `"true"` to [disable Tekton's built-in credential initialization](auth.md#disabling-tektons-built-in-auth) 
+and use Workspaces to mount credentials from Secrets instead.
+The default is `false`. For more information, see the [associated issue](https://github.com/tektoncd/pipeline/issues/3399).
```

Comment:
Suggest linking to the section of `auth.md` that describes this feature instead.  https://github.com/tektoncd/pipeline/blob/master/docs/auth.md#disabling-tektons-built-in-auth

Reply:
Hi @sbwsg ,

I linked that document above in the PR `[disable Tekton's built-in credential initialization](auth.md#disabling-tektons-built-in-auth) `

Is that what you are looking for?

Reply:
Oh haha 😅, sorry, I missed that.